src/main/java/co/lotc/lever/cmd/Countdown.java

```diff
@@ -0,0 +1,52 @@
+package co.lotc.lever.cmd;
+
+import static org.bukkit.ChatColor.*;
+
+import co.lotc.core.bukkit.util.Run;
+import co.lotc.core.command.annotate.Arg;
+import co.lotc.core.command.annotate.Default;
+import co.lotc.core.command.annotate.Flag;
+import co.lotc.core.command.annotate.Range;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import co.lotc.core.bukkit.util.LocationUtil;
+import co.lotc.lever.BaseCommand;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+public class Countdown extends BaseCommand {
+    public static final String PEX_COUNTDOWN_RADIUS = "lever.countdown.radius";
+    public static final Set<UUID> COOLDOWN_LIST = new HashSet<>();
+
+    @Flag(name="r",description="Countdown broadcast radius",type=double.class)
+    public void invoke(Player source, @Arg("Timer") @Default("5") @Range(min=5,max=60) int timer) {
+        if (COOLDOWN_LIST.contains(source.getUniqueId())) {
+            msg(RED + "Command on cooldown.");
```

Comment:
Use the validate() method that all the other commands use

Reply:
validate() is actually very bad practice when it comes to java readability :100: so its not mandatory to use it. But it'd be good to see what validate() is because it saves some boilerplate if you like it.

Reply:
It's not about readability here but consistency. But don't Most projects have some form of @NonNull or whatever